src/ssh/azext_ssh/custom.py

```diff
@@ -26,15 +26,14 @@
 from . import constants as const
 from . import resource_type_utils
 from . import target_os_utils
-
+from . import resource_tag_utils
 logger = log.get_logger(__name__)
 
 
 def ssh_vm(cmd, resource_group_name=None, vm_name=None, ssh_ip=None, public_key_file=None,
            private_key_file=None, use_private_ip=False, local_user=None, cert_file=None, port=None,
            ssh_client_folder=None, delete_credentials=False, resource_type=None, ssh_proxy_folder=None,
-           winrdp=False, yes_without_prompt=False, ssh_args=None):
-
+           winrdp=False, yes_without_prompt=False, ssh_args=None, resource_tag=None):
```

Comment:
resource_tag is a flag that can be true or false right? I think in that case it should be resource_tag=False. Correct me if I'm wrong.

Reply:
Resource tag has a passed in value of whatever characters. It is just a name to be searched in the resource tags to specify a port so not a flag. It can be specified for a specific resource tag name or the code should automatically look for "SSHPort" if not specified